core_weekly/github.py

```diff
@@ -1,37 +1,68 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
+from github import Github
+from github import UnknownObjectException
 import requests_cache
 import logging
 import requests
 import ssl
 import time
+from datetime import date, datetime, timedelta
 
 logger = logging.getLogger(__name__)
 ssl._create_default_https_context = ssl._create_unverified_context
 
 
-# Github api
+# Github API Client
 #
-class GitHub():
+class GitHubClient():
```

Comment:
Renaming to avoid confusion with PyGithub Github class

Reply:
There is no confusion, you can change the import class name
```python
from pygithub import GitHub as PyGitHub
```